UPGRADE-1.7.md

```diff
@@ -0,0 +1,423 @@
+# UPGRADE FROM 1.6 to 1.7
```

Comment:
If we want to be clean, to be merged in 2.0, and then merge it in 2.1

Reply:
Sorry for that :(